app/services/solidus_bolt/users/sync_addresses_service.rb

```diff
@@ -42,7 +41,7 @@ def convert_to_solidus(bolt_address)
           zipcode: bolt_address['postal_code'],
           name: bolt_address['name'],
           country: country,
-          phone: user_info['profile']
+          phone: bolt_address['phone_number'] || user_info['profile']['phone']
```

Comment:
Does it require a test?

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/104/commits/bd9e053d105a9e847f157c6ace38922d86ea3730